packages/webcert/src/components/question/AdministrativeQuestionPanel.tsx

```diff
@@ -17,7 +17,7 @@ interface StyledProps {
 }
 
 const Wrapper = styled.div<StyledProps>`
-  height: ${(props) => (props.shouldLimitHeight ? `calc(100% -  ${props.headerHeight}px);` : '100%;')}
+  height: ${(props) => (props.shouldLimitHeight ? `calc(100% -  ${props.headerHeight}px);` : '100%;')};
```

Comment:
Remove the ;

Reply:
Why should I remove the semicolon? It's the css syntax  
```css
selector {
  property: value;
}
```

Reply:
Sorry my bad I didn't see it was inside a styled component